src/Renderer/CliRenderer.php

```diff
@@ -92,9 +92,9 @@ public function __construct()
         }
 
         if (null === self::$terminal_width) {
-            if (!KINT_WIN && self::$detect_width) {
+            if (self::$detect_width) {
```

Comment:
Should this be `!$this->windows_output && self::$detect_width` instead? That'd guard against windows terminals that don't have `sapi_windows_vt100_support`

Reply:
I am investigating this. Will run a Windows 7 terminal for testing purposes. While I'm on it, I would like to also address the lack of colors on Windows, it seems to be related to this.